### PR TITLE
Replace jshint with eslint

### PR DESCRIPTION
**Pull request by ryan-codingintrigue**
Issue #96

Used the standard recommended [ESLint rules](http://eslint.org/docs/rules/). Only a few minor spacing changes, one quote change & moving a single `var` declaration outside of an `if` in order to comply.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "node": true
+  },
+  "globals": {
+    "describe": true,
+    "it": true,
+    "before": true
+  },
+  "rules": {
+    "indent": [2,2],
+    "quotes": [2,"single","avoid-escape"]
```
Comment (jackfranklin) [inline .eslintrc]: could we have a space after the commas here?

**Comment by jackfranklin** [comment]
Amazing, thank you! Just one teeny comment and then I'll merge. Much appreciated, thanks :)

**Comment by jackfranklin** [comment]
Thanks again!

**Comment by jackfranklin** [comment]
@ryan-codingintrigue you should make another PR that adds yourself to the contributors part of the package.json :)

**Comment by ryan-codingintrigue** [comment]
I will. Nice seeing this on Twitter as *yourfirstpr*,  I'm usually a bit awkward about submitting but this made it real PR-n00b-friendly, so thanks!

**Comment by jackfranklin** [comment]
@ryan-codingintrigue perfect, I'm so glad you think so and thank you for your contribution.

**Pull request merged**